README.md

```diff
@@ -416,14 +416,14 @@ const getMatchesFromEmbeddings = async (
       vector: embeddings,
       topK,
       includeMetadata: true,
-    });
-    return queryResult.matches || [];
+    })
+    return queryResult.matches || []
   } catch (e) {
     // Log the error and throw it
-    console.log("Error querying embeddings: ", e);
-    throw new Error(`Error querying embeddings: ${e}`);
+    console.log("Error querying embeddings: ", e)
+    throw (new Error(`Error querying embeddings: ${e}`,))
```

Comment:
I know it's what's currently in the code, but IMO this `,))`

```js
throw (new Error(`Error querying embeddings: ${e}`,))
``` 
has the look of a syntax error.

Reply:
I thought the same thing. It made me do a double take at the code itself. I'll fix in a follow-up.